src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -79,6 +81,7 @@ const ActiveState = ({
             {items.map((item: FeedItem) => {
                 const isFocused = item === activeEntry;
                 const refValue = isFocused ? activeFeedItemRef : undefined;
+                const fileVersionId = getProp(item, 'file_version.id');
```

Comment:
Maybe `itemFileVersionId` to be somewhat more clear that this is activity item-specific rather than file-specific?